.gitignore

```diff
@@ -13,6 +13,9 @@ hdfs-folders
 hadoop-*
 target/
 cpp/main
+data/code
+data/orc
```

Comment:
Actually data/orc should not be ignored, there's a test file there that is being used